src/processors/default/createERC721NFTsFromTransfersProcessor.ts

```diff
@@ -76,7 +79,8 @@ export const createERC721NFTsFromTransfersProcessor: (contracts: NftFactory[]) =
     trigger: newERC721Transfers(
       contracts
         .filter(c => c.standard === NFTStandard.ERC721) //only include ERC721 contracts
-        .map(c => ({ address: c.address, startingBlock: c.startingBlock! })) // map contracts to EthereumContract
+        .map(c => ({ address: c.address, startingBlock: c.startingBlock! })), // map contracts to EthereumContract
+      process.env.ETHEREUM_BLOCK_QUERY_GAP! // todo: overwrite to 2k?
```

Comment:
is todo still relevant?

Reply:
nope, I removed it. This is handled with the dynamic getLogs query now